src/main/java/org/alliancegenome/curation_api/services/Gff3Service.java

```diff
@@ -107,12 +118,13 @@
 	}
 	
 	@Transactional
-	public Map<String, List<Long>> loadAssociations(BulkLoadFileHistory history, Gff3DTO gffEntry, Map<String, List<Long>> idsAdded, BackendBulkDataProvider dataProvider, String assemblyId) throws ObjectUpdateException {
+	public Map<String, List<Long>> loadLocationAssociations(BulkLoadFileHistory history, ImmutablePair<Gff3DTO, Map<String,String>> gffEntryPair, Map<String, List<Long>> idsAdded, BackendBulkDataProvider dataProvider, String assemblyId, Map<String, String> geneIdCurieMap) throws ObjectUpdateException {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChA1MWFjZjljZDgyY2U4ZmExEgpjaGVja3N0eWxl -->